### ISSUE TITLE
check that all `setup.cfg` files are in sync across repos

### ISSUE THREAD

**Issue by perlinm** [SupertechLabs/general-superstaq#114]
Right now our `setup.cfg` files in `gss/qss/css/ss` are almost identical (I believe except for one line).  It would be nice to have a check that makes sure these files are in sync.  @alexabuko we can talk about how this might work.

**Comment by perlinm** [comment]
It looks like after #134, the only difference in `setup.cfg` across different repos will be a single line under `[coverage:run]`: `[gqc]ss/setup.cfg` have `omit = ./venv/*`, while `SuperstaQ/setup.cfg` has `omit = ./dev_tools/*`.

I think the `omit` line in SuperstaQ should be just removed, especially as it's non-functional.  Not sure about the `venv` one, although regardless of whether we keep it or toss it, we can make the choice identical across all repos.  That would make `setup.cfg` identical across all repos, which would greatly simplify this issue.  Having said that:
1. Whatever mechanism we use to ensure that `setup.cfg` is identical across all repos, we should build in a simple way of adding exceptions (otherwise it's a big :-1: for agility if and when we want to change a config in a single repo).  Maybe the checker could ignore lines with a special string (like, say, "# CUSTOM REPO CONFIG").
2. Do we want to keep the `venv` line in `[gqc]-ss`?  Not sure what everyone else's practices are here, but the disparity with SuperstaQ makes me suspect that this ignore is unnecessary?  @vtomole @richrines1

**Comment by vtomole** [comment]
Will have to think more about what you are saying here but right off the bat I agree that the `omit` in SuperstaQ can be removed.

**Comment by vtomole** [comment]
The `venv` one can be removed as well. Personally i don't run my `venv` in my root directory anymore. We can say on the dev guide that this is not allowed for us.

**Comment by vtomole** [comment]
> `# REPO-SPECIFIC CONFIG`

I like this as an escape hatch.
